src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -839,29 +840,35 @@ class AddItemsPage extends Component {
     const errors = {};
     errors.lineItems = [];
     const date = moment(this.props.minimumExpirationDate, 'MM/DD/YYYY');
+    const isPush = _.get(this.state.values.replenishmentType, 'name') !== REPLENISHMENT_TYPE_PUSH;
```

Comment:
Why it's swapped? If `name !== PUSH`, then `isPush`? And the same for pull. Why it cant be `isPush if name === Push`?

Reply:
you are right, this was my mistake